xline-test-utils/src/lib.rs

```diff
@@ -197,6 +213,63 @@ impl Cluster {
         let decoding_key = DecodingKey::from_rsa_pem(public_key).unwrap();
         Some((encoding_key, decoding_key))
     }
+
+    pub fn default_cluster_config() -> ClusterConfig {
```

Comment:
We can derive Debug trait for `ClusterConfig` rather than implement it manually.

Reply:
Fixed